tensorrt_llm/_torch/auto_deploy/custom_ops/flashinfer_attention.py

```
@@ -393,11 +461,26 @@ def _init_workspace(si: SequenceInfo) -> torch.Tensor:
             buffer = torch.empty(320 * 1024 * 1024, dtype=torch.uint8, device=si.device)
-            cls._get_planner().init_workspace(buffer)
+            cls._get_planner().init_workspace(buffer, si.max_batch_size, si.num_pages)
             return buffer
 
         return {"workspace_buffer": _init_workspace}
 
+    @classmethod
+    def host_prepare_for_forward(cls, sequence_info: SequenceInfo):
+        print("DEBUG: host_prepare_for_forward called for flashinfer attention")
+        batch_info = sequence_info._input_buffer.get_host_view("batch_info")
+        num_prefill, num_prefill_tokens, num_decode = batch_info.tolist()
```

Ruff (0.14.10)

Unpacked variable `num_prefill_tokens` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)